lib/node-package-manager.ts

```diff
@@ -48,30 +24,54 @@ export class NodePackageManager implements INodePackageManager {
 			}
 
 			try {
-				return this.loadAndExecute("install", [pathToSave, packageName], { config: config }).wait();
+				let jsonContentBefore = this.$fs.readJson(path.join(pathToSave, "package.json")).wait();
+				let dependenciesBefore: Array<string> = [];
+				if(jsonContentBefore && jsonContentBefore.dependencies) {
+					for(let name in jsonContentBefore.dependencies) {
+						dependenciesBefore.push(name);
+					}
+				}
+
+				let flags = this.getFlagsString(config, true);
+				let params = ["install"];
+				if(packageName !== pathToSave) {
+					params.push(packageName); //because npm install ${pwd} on mac tries to install itself as a dependency (windows and linux have no such issues)
+				}
+				params = params.concat(flags);
+				this.$childProcess.spawnFromEvent("npm", params, "close", { cwd: pathToSave }).wait();
```

Comment:
To be more precise in the explanation:
 - on Windows, the default command prompt is able to resolve npm to `npm.cmd` when you execute `npm install`
 - childProcess.spawn method does not launch new shell, it just starts executable in PATH (or passed with full path)

In the current case, childProcess is trying to start a new process called `npm` but it cannot be resolved as the real name of the executable on Windows is `npm.cmd`.
We have such code, you can take a look at it [here](https://github.com/telerik/mobile-cli-lib/blob/79633174cb36302a0ec4c1eecc382ff45f7898ed/appbuilder/services/npm-service.ts#L51-L61)